jvector-base/src/main/java/io/github/jbellis/jvector/graph/OnHeapGraphIndex.java

```diff
@@ -91,7 +96,7 @@ public void addNode(int node) {
 
   /** must be called after addNode once neighbors are linked in all levels. */
   void markComplete(int node) {
-    entryPoint.accumulateAndGet(
+    entryPointUpdater.accumulateAndGet(this,
```

Comment:
I believe this is equivalent to the old code, the reason to use an Updater is when you have multiple long fields and you want to save the memory overhead of wrapping each in an Atomic

Reply:
It failed for me which is why I changed it

Reply:
I will try and undo this in #113 closing this one